test/built-ins/Temporal/PlainDateTime/prototype/with/cases.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: A variety of "normal" (non-throwing, non-boundary case, non-null, etc.) arguments
+esid: sec-temporal.plaindatetime.prototype.with
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const datetime = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, 30, 123, 456, 789);
+
+assert.sameValue(
+  `${datetime.with({ year: 2019 })}`,
+  "2019-11-18T15:23:30.123456789",
+  "with year works"
+);
```

Comment:
I would also prefer `TemporalHelpers.assertPlainDateTime` over comparing stringifications here (and in general, when not specifically testing stringification).

Reply:
done

Reply:
Doesn't seem to have made it

Reply:
Looks like I accidentally included `temporalHelpers.js`. Dropped now.